### ISSUE TITLE
AttributeError: 'MaskRCNN' object has no attribute 'prop_head'

### ISSUE THREAD

**Issue by Allamrahul** [anirudh-chakravarthy/ObjProp#4]
Hi,

I am using the masktrackrcnn pretrained .pth file and masktrack_rcnn_r50_fpn_1x_youtubevos.py file. I am able to train the model but inference is failing due to the above reason. I see that prop_head is indeed an attribute but I am unsure why its complaining.

**Comment by anirudh-chakravarthy** [comment]
Hi,

I am not sure why this is happening. Can you share the error trace so I can have a look?

**Comment by Allamrahul** [comment]
We tried to use the masktrack_rcnn_r50_fpn_1x_youtubevos.py config to run just the masktrackrcnn. We tried to setup the base repository for masktrackrcnn repository (https://github.com/youtubevos/MaskTrackRCNN.git) but we keep facing issues since the torch version being used in that repository is pretty old. Hence, we tried to setup just masktrackrcnn, without the propagation head and thats when we faced that issue. 

I am closing this because we have been able to build the masktrackrcnn repo separately. Hence, we are not longer pursuing this avenue. Hence, closing this issue.
